src/browser/api/window.js

```diff
@@ -502,6 +502,28 @@ Window.create = function(id, opts) {
             }
         });
 
+        const emitToAppAndWin = (...types) => {
+            let isMainWindow = (uuid === name);
+            types.forEach(type => {
+                ofEvents.emit(route.window(type, uuid, name), { topic: 'window', type, uuid, name });
+                if (isMainWindow) {
+                    ofEvents.emit(route.application(`window-${type}`, uuid), { topic: 'application', type, uuid, name });
```

Comment:
My understanding here is that if it is the main window that it should propagate as an application event, if it is a child window it should be with the `window-` prefix. Is that correct? @rdepena @StevenEBarbaro @joneit